src/Classes/Particle.ts

```diff
@@ -170,60 +170,95 @@ export class Particle {
         return res;
     }
 
+    public resetVelocity() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'resetVelocity' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5141273)